CONTRIBUTING.md

```diff
@@ -12,10 +12,10 @@ To add your application to the [DigitalOcean Marketplace](https://marketplace.di
 1. Create a git branch that includes the name of your app (example: `$APP_NAME-first-pr`)
 1. Run the following commands to create your `deploy.sh`, `upgrade.sh`, `uninstall.sh` and `values.yml` files. They will be located in `stacks/$APP_NAME`:
 ```bash
-export HELM_REPO_NAME=stable
-export HELM_REPO_URL=https://charts.helm.sh/stable
+export HELM_REPO_NAME=$REPO_NAME
+export HELM_REPO_URL=https://charts.helm.sh/your-helm-repo

```

Comment:
Maybe we want this to not assume the repo is located at `https://charts.helm.sh/` as many vendors now keep their charts on github page.  perhaps `$REPO_URL` would be better?

Reply:
Yes, good point!